source/common/grpc/common.cc

```diff
@@ -53,8 +53,8 @@ void Common::chargeStat(const Upstream::ClusterInfo& cluster, const std::string&
   chargeStat(cluster, "grpc", grpc_service, grpc_method, success);
 }
 
-Optional<Status::GrpcStatus> Common::getGrpcStatus(const Http::HeaderMap& trailers) {
```

Comment:
Yep - the `grpc-status` header is provided in the trailers in gRPC http://www.grpc.io/docs/guides/wire.html